Tests/DistributedActorsTests/Plugins/ClusterSingleton/ClusterSingletonPluginTests.swift

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Actors open source project
+//
+// Copyright (c) 2019-2022 Apple Inc. and the Swift Distributed Actors project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of Swift Distributed Actors project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import DistributedActors
+import DistributedActorsTestKit
+import XCTest
+
+final class ClusterSingletonPluginTests: ClusterSystemXCTestCase {
+    func test_singletonPlugin_clusterDisabled() async throws {
+        throw XCTSkip("!!! Skipping test \(#function) !!!") // FIXME(distributed): disable test until https://github.com/apple/swift-distributed-actors/pull/1001
+
+        // Singleton should work just fine without clustering
+        let test = await setUpNode("test") { settings in
+            settings.enabled = false
+            settings += ClusterSingletonPlugin()
+        }
+
+        // singleton.host
+        let ref = try await test.singleton.host(of: TheSingleton.self, name: TheSingleton.name) { actorSystem in
+            TheSingleton(greeting: "Hello", actorSystem: actorSystem)
+        }
+        let reply = try await ref.greet(name: "Charlie")
+        reply.shouldStartWith(prefix: "Hello Charlie!")
+
+        // singleton.ref (proxy-only)
+        let proxyRef = try await test.singleton.proxy(of: TheSingleton.self, name: TheSingleton.name)
```

Comment:
Impl wise I actually realized that we don't handle this well, we might just do the host version for now.

Reply:
👌